api/src/main/java/com/ford/labs/retroquest/metrics/Metrics.java

```diff
@@ -24,15 +28,54 @@ public int getTeamCount() {
         return teamRepository.findAll().size();
     }
 
+    public int getTeamCount(LocalDate startDate, LocalDate endDate) {
+        LocalDate finalStartTime = startDate == null ? LocalDate.MIN : startDate;
+        LocalDate finalEndTime = endDate == null ? LocalDate.now() : endDate;
+        return teamRepository.findAll().stream()
+                .filter(team -> !team.getDateCreated().isBefore(finalStartTime))
+                .filter(team -> !team.getDateCreated().isAfter(finalEndTime))
+                .collect(toList()).size();
+    }
+
     @ManagedAttribute
     public int getFeedbackCount() {
         return feedbackRepository.findAll().size();
     }
 
+    int getFeedbackCount(LocalDate startTime, LocalDate endTime) {
+        LocalDateTime finalEndTime = endTime == null ? LocalDateTime.now() : endTime.atStartOfDay();
+        LocalDateTime finalStartTime = startTime == null ? LocalDateTime.MIN : startTime.atStartOfDay();
+        List<Feedback> feedbackList =  feedbackRepository.findAll().stream()
```

Comment:
This filtering can also be done with Spring Data JPA - https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.details

Doing it this way would be less code all around with less to test.

- Lee D